packages/story-editor/src/components/library/panes/pageTemplates/savedPageTemplate.js

```diff
@@ -262,27 +232,32 @@ function PageTemplate(
         >
           <Icons.TrashFilledSmall />
         </DeleteButton>
-      </PreviewPageWrapper>
 
-      {page.title && (
-        <PageTemplateTitle isActive={isActive}>{page.title}</PageTemplateTitle>
-      )}
+        {page.title && (
+          <PageTemplateTitleContainer isActive={isActive}>
+            <Text
+              as="span"
+              size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}
+            >
+              {page.title}
+            </Text>
+          </PageTemplateTitleContainer>
+        )}
+      </PreviewPageWrapper>
```

Comment:
Just curious, how does one get a title added to custom saved template? I mocked it while developing to make sure styles were applied correctly. But I am unsure how a user would name a template.

Reply:
@miina do you know this? 

I feel like it can't? There's no way to title saved templates.

Reply:
Same here -- as far as I know there's no way to do that currently.

Reply:
yeah, this can probably just get removed then, no need to include it since saved templates don't have titles. if they ever do we can add it in then.